src/services/DhcService.ts

```diff
@@ -359,6 +396,14 @@ export class DhcService extends DisposableBase implements IDhcService {
     await saveRequirementsTxt(dependencies);
   }
 
+  async deleteVariable(variableDefinition: VariableDefintion): Promise<void> {
+    if (this.session == null) {
+      return;
+    }
```

Comment:
Why don't we throw an error in these cases if the session is null when trying to execute one of these commands? This is a situation that should not happen, correct?